docs/adr/005-Use-HTML-sanitizer-to-avoid-XSS-attacks.md

```diff
@@ -0,0 +1,83 @@
+# 005. Use HTML sanitizer to avoid XSS attacks
+
+Date: 29-no-2021
+
+## Status
+
+PROPOSED  
+
+## Summary
+
+*CKEditor4 team detected an XSS vulnerability on the Mathtype devkit package. A user could inject malicious to a website code encaplusating them inside a <math> tag.*
```

Comment:
I would omit that the CKEditor4 team found the vulnerability, and add that we have a XSS vulnerability since we don't review if the code obtained is correct, or something similar

Reply:
It's makes sense, I have modified this part.